src/vl-upload.js

```diff
@@ -384,4 +402,23 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
   _enableAutoProcessQueue() {
     this._dropzone.options.autoProcessQueue = true;
   }
+
+  _addSlots() {
+    Promise.all([this.uploadElement]).then(() => {
+      const subTitleSlotNodes = this._subTitleSlotElement.assignedNodes();
+      if (subTitleSlotNodes.length > 0) {
+        const small = this.uploadElement.querySelector('small');
+        small.innerHTML = subTitleSlotNodes[0].outerHTML;
+      }
+
+      const titleSlotNodes = this._titleSlotElement.assignedNodes();
+      if (titleSlotNodes.length > 0) {
+        const buttonContainer = this.uploadElement.querySelector('.vl-upload__element__button__container');
+        buttonContainer.innerHTML = titleSlotNodes[0].outerHTML;
+      }
+    }).finally(() => {
+      this._titleSlotElement.remove();
+      this._subTitleSlotElement.remove();
+    });
+  }
```

Comment:
Wat je hiermee denk ik nog niet mee hebt dan, is als er aan het slot iets zou veranderen. Typisch wordt dat opgelost met een MutationObserver, maar misschien is het als initiele versie goed genoeg om alleen 'at load time' de configuratie 'over te kopieren' (tenminste, als ik goed begrijp dat dat is wat er gebeurt)

Reply:
ja klopt wel, zag dat in de vl-alert daar wel een test voor was. Nu zoals je zegt weet niet of we dat in eerste instantie al nodig hebben maar indien wel. Moet ik eens bekijken of we dat niet anders kunnen oplossen.